.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
This should be part of your global gitignore, not of a particular project. Since it is specific to your local setup, not to this project. (related article https://thomashunter.name/blog/global-gitignore-vs-repository-gitignore/)